fundraiser/src/utils.sw

```diff
@@ -0,0 +1,17 @@
+library utils;
+
+dep errors;
+
+use std::{assert::require, chain::auth::{AuthError, msg_sender}, identity::Identity, result::*};
+
+use errors::UserError;
+
+pub fn sender_identity() -> Identity {
+    // TODO: use msg_sender().unwrap() (and remove this function) when that becomes functional
+    let sender: Result<Identity, AuthError> = msg_sender();
+    sender.unwrap()
+}
+
+pub fn validate_id(id: u64, count: u64) {
```

Comment:
Is there a reason why you made this require check its own function and the other require checks not?

Reply:
Keeping the code DRY (don't repeat yourself). The other checks do not get used over and over. This one does. It makes it easier to manage if it were to be changed.